.github/workflows/test_python.yml

```diff
@@ -60,8 +65,17 @@ jobs:
     - run: buildozer init
     - name: SDK, NDK and p4a download
       run: |
-        sed "s/# android.accept_sdk_license = False/android.accept_sdk_license = True/" -i buildozer.spec
+        sed -i.bak "s/# android.accept_sdk_license = False/android.accept_sdk_license = True/" buildozer.spec
         buildozer android p4a -- --help
+    # Install OS specific dependencies
+    - name: Install Linux dependencies
+      if: matrix.os == 'ubuntu-latest'
+      run: sudo apt -y install automake
+    - name: Install macOS dependencies
+      if: matrix.os == 'macOS-latest'
+      run: |
+        brew install automake
+        sudo ln -sfn /usr/local/opt/openssl /usr/local/ssl
```

Comment:
This is a workaround that could easily be integrated p4a side by patching Python to add `/usr/local/opt/openssl` to the search path.
I actually think it should be a CPython upstream fix, I may look into this in the future

Reply:
That's interesting indeed. Did macOS move this recently? Anyway, seems like a good fix for CI purposes.

Is this something users are going to have problems with?

Reply:
I assumed user may have problem with it, even though I didn't directly see the problem yet on my macOS.
What I saw is without this fix the python3 `config.log` clearly show it didn't find the openssl lib. However later I didn't get the pip SSL error. However on the CI I do get the error :thinking: 
So yes I may pull request this CPython upstream because the fix is easy